src/modules/translation/defaults/en.json

```diff
@@ -44,6 +44,18 @@
         "mobile": "You can use mobile browsers such as {coinbase_link} or {imtoken_link}.",
```

Comment:
You think we should update these options? the sign in page still reads 
```json
"desktop": "You can use the {metamask_link} extension or a hardware wallet like {ledger_nano_link}.",
```
@cazala

Reply:
good catch, i think we could update it to something like `You can use the {metamask_link} extension, a hardware wallet like {ledger_nano_link} or login using your email account via {fortmatic_link}.` wdyt?